extensions/guacamole-auth-jdbc/modules/guacamole-auth-jdbc-mysql/src/main/resources/guac-manifest.json

```diff
@@ -21,7 +21,8 @@
     "translations" : [
         "translations/en.json",
         "translations/fr.json",
-        "translations/ru.json"
+        "translations/ru.json",
+        "translations/es.json"
```

Comment:
Please maintain alphabetical order of the files listed (same in the other pair of `guac-manifest.json`).